Misc/NEWS.d/next/Core and Builtins/2024-04-13-18-59-25.gh-issue-115874.c3xG-E.rst

```diff
@@ -0,0 +1 @@
+Fixed a possible segfault during garbage collection of ``_asyncio.FutureIter`` objects
```

Comment:
Minor: This should have been in Library section rather than core builtins, its a bug of asynio library.

Reply:
I've opened a PR to address these comments in #121638